pyszuru/__init__.py

```diff
@@ -31,19 +39,43 @@ def getTag(self, id_: str) -> Tag:
 
     def createTag(self, name: str) -> Tag:
         # Get default tag category
-        tag_cats = [
-            x["name"]
-            for x in self._call("GET", ["tag-categories"])["results"]
-            if x["default"]
-        ]
-        assert len(tag_cats) == 1
-        default_cat = tag_cats[0]
+        tag_cats = self._call("GET", ["tag-categories"])["results"]
+        default_cat = [x for x in tag_cats if x["default"]]
+        assert len(default_cat) == 1
+        default_cat = default_cat[0]["name"]
+
         # Create and return tag
         t = Tag(self, {})
         t._json_new = {"names": [name], "category": default_cat}
         t.push()
         return t
 
+    def getPool(self, id_: int) -> Pool:
+        p = Pool(self, {"id": id_})
+        p.pull()
+        return p
+
+    def createPool(
+        self,
+        name: "str | List[str]",
+        category: str = None,
+        posts: List[Post] = None,
+        description: str = None,
```

Comment:
These arguments aren't being put anywhere.

Reply:
Removed to be consistent with the other resource factory methods.

I think it's more logical to create it with default values, then update those values later on.